devicon.json

```
@@ -1929,6 +1929,25 @@
         "color": "#59B3D8",
         "aliases": []
     },
+    {
+        "name": "couchbase",
+        "altnames": [],
+        "tags": [
+            "database",
+            "nosql",
+            "cloud"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "original"
+            ]
+        },
+        "color": "#EA2328",
+        "aliases": []
```

Since this icon can also be the plain icon, we should add an alias from original to plain. You can read more about it under point 3. here: https://github.com/devicons/devicon/wiki/SVG-Versions
> If the icon can be used as a font it should also be given the appropriate [aliases](https://github.com/devicons/devicon/wiki/Updating-%60devicon.json%60#aliases-and-aliasobj).
In the case of Apple, you should create an [alias object](https://github.com/devicons/devicon/wiki/Updating-%60devicon.json%60#aliases-and-aliasobj) where the base is original and the alias is plain
 
```suggestion
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            },
        ]
```

---

Sir, I have added the required alias as well as the wordmark version of the SVG. Please review the changes

---

Hi!
Unfortunately the wordmark version does not mert the requirement for a plain icon. One of the requirements is that it should be mono-colored.
To fix this, simply add another icon called couchdb-plain-wordmark.svg and remove the alias object from devicon.json

---

You can read more about the version requirements here:
- [SVG-Versions](https://github.com/devicons/devicon/wiki/SVG-Versions)
- [SVG-Standards](https://github.com/devicons/devicon/wiki/SVG-Standards)

---

Sorry but I am unable to understand your concern. In my latest commits, I have added `couchbase-original-wordmark.svg` and NOT `couchbase-plain-wordmark.svg`

As per the requirements mentioned, the plain one should be a one-color version but the original-wordmark says that it should be similar to original along with name. Hence, the original and original-wordmark can contain multiple colors as in my pull request. 

For example, I can show you couchdb wordmark version [couchdb-original-wordmark](https://github.com/devicons/devicon/blob/master/icons/couchdb/couchdb-original-wordmark.svg
)
Similarly mine is for Couchbase as you can see [couchbase-original-wordmark](https://github.com/devicons/devicon/pull/1932/commits/e94f34c5ba5218abf57892889ea17ec026e06402)

I have also added `couchbase-plain.svg` and `couchbase-plain-wordmark.svg`

According to requirements, the plain SVG can be in original color, all black or all white. I have decided to use black color for both of them.

Thank You

---

I added new suggestions. See if you understand those better :)